server/src/controllers/PayController.js

```diff
@@ -0,0 +1,52 @@
+const PayServices = require('../services/PayServices');
+const stripeSecretKey = process.env.STRIPE_SECRET_KEY || require('../config/secretKeys').stripeSecretKey;
+
+const stripe = require("stripe")('sk_test_Hrs6SAopgFPF0bZXSN3f6ELN');
+const express = require('express');
+const app = express();
+app.use(express.static('public'));
+
+const YOUR_DOMAIN = 'http://localhost:5000';
+
+
+const PayController = {
+    // may not be needed? 
+    async validation(req, res, next) {
+      const froshData = req.body;
+
+      try {
+          await PayServices.validateEmail(froshData);
+          await PayServices.validateInformation(froshData);
+
+          res.status(200).send({ message: 'Successfully validated frosh!' });
+      } catch (e) {
+          next(e);
+      }
+    },
+
+    async getPrice(req, res, next) {
+        const listOfFrosh = await Frosh.find();
+        const amount = listOfFrosh.length <=99 ? '56.99' :'61.99'; // need to change prices and early bird??
+        res.status(200).send(amount); 
+    },
+    
+    async paymentIntent(req, res, next) {
+      const { items } = req.body;
+    
+      // Create a PaymentIntent with the order amount and currency
+      const paymentIntent = await stripe.paymentIntents.create({
+        amount: calculateOrderAmount(items),

```

Comment:
For now, can you implement a `calculateOrderAmount` function here that just returns a static value of '60.00' ? We can update this later once we know the final pricing rules.

Reply:
yep for sure